docs/keywords.md

```diff
@@ -0,0 +1,53 @@
+
+Keyword | Description
+------- | -----------
+**Type of Linguistic Change** |
+  `morphological` | character level or inflectional/morphological changes
+  `lexical` | the transformation substitutes different words or phrases
+  `syntactic` | the transformation changes the syntax of the input (eg. active to passive)
+  `word-order` | if the order of the words or phrases is changed (eg. topicalisation, changing the order of semantic roles)
+  `discourse` | operating on or generating units of language longer than a sentence (eg. paragraph, multiple utterances in a dialog setting, etc)
+  `noise` | The generation adds random noise which would perturb the examples (eg. removing punctuations, changing case, repeating words/characters)
+**Type of algorithm** |
+  `rule-based` | if the implementation uses a heuristic or rules based approach
+  `model-based` | if the implementation uses a machine learning model
+  `api-based` | if the implementation uses an external api or tool
+  `external-knowledge-based` | if the implementation uses a corpora, a separate knowledge base, etc.
+  `transformer-based` | if the implementation uses a transformer based model (eg. BERT, T5)
+  `parser-based` | if the implementation uses any syntactic (dependency, constituency, etc) or shallow semantic parser or other semantic parsers (eg. semantic role labelling)
+  `tokenizer` | if the implementation uses a tokenizer
+  `chunker` | if the implementation uses a chunker
+  `discourse-segmenter` | if the implementation uses a discourse segmenter
+**The naturalness of the generation** |
+  `unnatural-sounding` | if the generations might not sound natural eg. if synthesized to speech, would they not sound natural as their source sentences. Eg, adding noun definitions might make the sentence factually correct but people might not use the same language while speaking.  
+  `unnaturally-written` | generations in written format which might not look natural to a reader. homophonic translations, filler words
+**The potential accuracy & precision of the generation** | 
+  `high-precision` | The transformation generates what it intends to generate
+  `low-precision` | The transformation sometimes does not generate what it intends to generate (eg. often changes meaning)
+  `low-coverage` | The transformation might not return an output for all types of inputs but only a handful of inputs
+  `high-coverage` | The transformation can return an output for almost all types of inputs
+  `high-generations` | The transformation generates a lot of (possibly diverse) outputs for a single input (eg beam search or substitutions with large number of alternate choices)
+  `low-generations` | The transformation generates few outputs for a single input (substitution with lesser number of alternate choices)
```

Comment:
Switch the order of `high-generations` and `low-generations`?

Reply:
made it consistent for all.